ui/src/components/RadioButton/RadioButton.jsx

```diff
@@ -0,0 +1,60 @@
+import classNames from 'classnames'
+import styles from './RadioButton.module.scss'
+import PropTypes from 'prop-types'
+
+/**
+ * ### example: <br/>
+ * ```
+ * import RadioButton from './components/RadioButton/RadioButton'
+ *
+ *<RadioButton
+ *  id="radioInput"
+ *  labelName="Radio 1"
+ *  inverted
+ *  required
+ *  />
+ *
+ *
+ *
+ *
+ * ```
+ * ### props: <br/>
+ * - id - identificator of input element - it will connect with label to improve **a11y**<br/> isRequired: true<br/>default value: "radioInput"
+ * - labelName - value for label to improve readable input<br/> isRequired: true<br/>default value: "Radio 1"
+ * - required - set if user must fill this input or it's optional<br/> isRequired: false<br/>default value: false
+ * - inverted - set layout order - first label then input or reversed <br/> isRequired: false<br/>default value: false
+ */
+
+const RadioButton = ({ labelName, id, isRequired, inverted }) => {
```

Comment:
Przydałaby się jeszcze jakiś props `value`, która określałby wartość w formularzu dla Radio Buttona (jeżeli programista nie poda tej wartości, to domyślnie może się równać `labelName`).

Reply:
W przyszłości moglibyśmy chcieć też z komponentu rodzica wykonywać konkretną akcję przy zmianie statusu (zaznaczony/niezaznaczony) radio buttona (np. możemy chcieć zaktualizować stan). W tym celu przydałaby się w propsach jakaś dodatkowa wartość, która by za to odpowiadała, np. `onSelected`, która przyjmowałaby funkję wykonywaną na `onChange` dla `<input type="radio"`

Reply:
Bezpiecznym rozwiązaniem byłoby wykorzystanie `spread operatora` dla reszty propsów w taki sposób: 
```suggestion
const RadioButton = ({ labelName, id, isRequired, inverted, ...props }) => {
```

W tym momencie pod zmienną `props` kryją się pozostałe nieuwzględnione wcześniej propy, które programista mógłby chcieć dodać. Możesz je wtedy dodać do definicji inputu w ten sposób: 

```jsx
<input
    id={id}
    className={classNames(styles.radioButton__value)}
    type="radio"
    required={isRequired}
    {...props}
  />
```

Reply:
I wtedy już nie trzeba uwzględniać ani `value`, ani `onChange` 😉

Reply:
I've added two additional props for now to take control on it which props are passed to component ;). Thanks for knowledge sharing :D